sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/PartitionReaderFactory.java

```diff
@@ -55,7 +55,7 @@ default PartitionReader<ColumnarBatch> createColumnarReader(InputPartition parti
   }
 
   /**
-   * Returns true if the given {@link InputPartition} should be read by Spark in a columnar way.
+   * @return true if the given {@link InputPartition} should be read by Spark in a columnar way.
    * This means, implementations must also implement {@link #createColumnarReader(InputPartition)}
```

Comment:
and here

Reply:
> and here

Thank you very much for review and sorry for the late response. I will try to modify the code according to your suggestions